src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/ActiveLicense.cs

```diff
@@ -20,15 +20,22 @@ public static ActiveLicenseFindResult Find(string applicationName, params Licens
             var licenseSourceResultToUse = LicenseSourceResult.DetermineBestLicenseSourceResult(results.ToArray());
             if (licenseSourceResultToUse != null)
             {
-                activeLicense.Report.Add($"Selected active license from {licenseSourceResultToUse.Location}");
+                var selectedLicenseReportItem = $"Selected active license from {licenseSourceResultToUse.Location}";
+                activeLicense.Report.Add(selectedLicenseReportItem);
```

Comment:
It's just that I didn't see the benefit of making that kind of change to be worth requiring the other projects consuming the package to have to change as well.